js/main.js

```diff
@@ -389,7 +393,7 @@ var mapView = {
     switch (menu) {
       case 1:
         var current_user_stats = self.user_data[self.settings.users[user_id].username].stats[0].inventory_item_data.player_stats;
-        $('#subtitle').html('Trainer Info');
+        $('#subtitle').html(self.settings.users[user_id]);
```

Comment:
I readded your changes to the username display. I must have overlooked this line while merging. 
Additionally I observed that 'pokemon_caught_by_type' is not present for freshly created users and my pikachu lookup in the pokedex failed if it hadn't been caught yet. I fixed those to errors, too.